tide_landing_page.install

```diff
@@ -384,3 +384,22 @@ function tide_landing_page_update_8008() {
     $field_config->save();
   }
 }
+
+/**
+ * Summary field in carousal Card event is restricted to 200 characters.
+ */
+function tide_landing_page_update_8009() {
+  /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $entity_form_display */
+  $entity_form_display = Drupal::entityTypeManager()
+    ->getStorage('entity_form_display')
+    ->load('paragraph.card_event.default');
+  $field_paragraph_summary = $entity_form_display->getComponent('field_paragraph_summary');
+  if (!$field_paragraph_summary['third_party_settings']) {
```

Comment:
@vincent-gao I think if `['maxlength']['maxlength_js']` is set there shouldn't be any forced change, so I suggest extending this conditional statement to perform a check of adding a check in this conditional for `['maxlength']['maxlength_js']` and removing the `elseif`

Reply:
thanks @GROwen , I added logic for checking if the `maxlength_js` is empty. if yes, i will force `maxlength_js` to 200.